23_administration/991_monitoring.txt

```diff
@@ -0,0 +1,86 @@
+=== Monitoring
+
+This section will give recommendations for:
+
+* Monitoring Rudder itself (besides standard monitoring)
+* Monitoring the state of your configuration management
+
+==== Monitoring Rudder itself
+
+===== Monitoring a Node
+
+The monitoring of a node mainly consists in checking that the Node can speak with
+its policy server, and that the agent is run regularly.
+
+A good place to start is to check the content of the last run log file.
+It can be found in '/var/rudder/cfengine-community/output/previous'.
+
+You can search lines containing 'FATAL:', 'Fatal :', or 'Could not get an updated configuration'
+with a log monitoring tool to automatically detect communication issues with the policy server.
+
+To get the last run time, you can lookup the modification date of '/var/rudder/cfengine-community/output/previous'.
```

Comment:
I think we should recommend to check the "/var/rudder/cfengine-community/last_successful_inputs_update" file instead. It is updated whenever we try to update promises, not just when the agent is run, and therefore tests more things.

Also, it is easier to "fix" - if your monitoring tool is in alert, and you run "rudder agent run" or "rudder agent update" and fix the problem, it will immediately end the alert, whereas with /var/rudder/cfengine-community/output/previous you'd have to wait for the next cf-execd scheduled run (because other runs don't touch this file).